var/spack/repos/builtin/packages/py-packaging/package.py

```diff
@@ -24,6 +24,8 @@ class PyPackaging(PythonPackage):
 
     depends_on("py-flit-core@3.3:", when="@22:", type="build")
 
+    depends_on("python@3.7:", when="@22:")
```

Comment:
type=build/run

Reply:
Also can you add a comment saying why this is necessary so I don't forget and remove it?